context/app/static/js/index.css

```diff
@@ -28,3 +29,18 @@ li {
     font-weight: bolder;
     font-size: 2.6rem;
 }
+
+/* https://css-tricks.com/couple-takes-sticky-footer/ */
+.main-content {
+    min-height: calc(100vh - 120px);
+}
+
+.container-lg {
+    max-width: 1240px;
```

Comment:
Did Nils ask for this? I feel like with the visualizations we want to do, if someone makes their window absurdly wide, perhaps we should respect their intentions.

Reply:
Yes, but we didn't specifically speak about the visualizations. I'd agree with you on it and the  tool tabs can be handled separately with wise.